lib/screens/admin_home_screen.dart

```diff
@@ -1,47 +1,51 @@
 import 'package:cobe_hive_mobile_app/app_colors.dart';
+import 'package:cobe_hive_mobile_app/providers/leave_request_provider.dart';
 import 'package:cobe_hive_mobile_app/widgets/app_header.dart';
 import 'package:cobe_hive_mobile_app/widgets/chip_list.dart';
 import 'package:cobe_hive_mobile_app/widgets/employee_card_list.dart';
-import 'package:cobe_hive_mobile_app/widgets/leave_request_list.dart';
+import 'package:cobe_hive_mobile_app/widgets/admin_screen_request_list.dart';
 import 'package:cobe_hive_mobile_app/widgets/search_bar.dart';
 import 'package:flutter/material.dart';
 
 import 'package:cobe_hive_mobile_app/widgets/expandable_fab.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
 
-class AdminHomeScreen extends StatelessWidget {
+class AdminHomeScreen extends ConsumerWidget {
   const AdminHomeScreen({super.key});
 
   @override
-  Widget build(BuildContext context) {
+  Widget build(BuildContext context, WidgetRef ref) {
     return GestureDetector(
       onTap: () => FocusManager.instance.primaryFocus?.unfocus(),
-      child: const Scaffold(
+      child: Scaffold(
         body: Column(
           children: <Widget>[
-            AppHeader(),
-            Padding(
+            const AppHeader(),
+            const Padding(
               padding: EdgeInsets.only(left: 18, right: 18, top: 25),
               child: _ManageRequestsBar(),
             ),
             SizedBox(
               height: 150,
-              child: LeaveRequestList(),
+              child: AdminScreenRequestList(
+                leaveRequestList: ref.watch(leaveRequestListPendingProvider),
```

Comment:
This way the whole screen changes when you add a request to the pending list.
You should `ref.watch(leaveRequestListPendingProvider)` inside `AdminScreenRequestList `.